server/src/test/java/io/crate/integrationtests/CursorITest.java

```diff
@@ -78,13 +78,39 @@ public void test_declare_fetch_and_close() throws Exception {
         }
     }
 
+    @Test
+    public void test_fetching_backwards_from_cursor_positioned_at_start_returns_empty_result() throws Exception {
```

Comment:
Fine to keep the integration test, but in case you aren't aware we also have cheaper unit tests to test the cursor fetch. For example: 

https://github.com/crate/crate/blob/7a6f19bd0d566a5767aa72f82949231b24246e80/server/src/test/java/io/crate/action/sql/CursorTest.java#L81-L103

Reply:
I thought so, but I'm tending a bit to keep iTest, also happy to convert to unit test though, if you prefer.

Reply:
Converted to unit test